apps/namada-interface/src/App/AccountOverview/AccountOverview.tsx

```diff
@@ -27,11 +27,34 @@ import {
 } from "./AccountOverview.components";
 import { DerivedAccounts } from "./DerivedAccounts";
 
+import { atom, useAtomValue, useSetAtom } from "jotai";
+import { loadable } from "jotai/utils";
+import { accountsAtom, balancesAtom } from "slices/accounts";
+import { balanceToFiatAtom } from "slices/coins";
+import { fiatCurrencyAtom } from "slices/settings";
+
 //TODO: move to utils when we have one
 const isEmptyObject = (object: Record<string, unknown>): boolean => {
   return object ? Object.keys(object).length === 0 : true;
 };
 
+const totalBalanceAtom = atom(async (get) => {
```

Comment:
Is this atom defined here because it is specific to this component? :) Asking because most of those are in the slices

Reply:
Yes, just because it is specific to this component, but I can move it :smile:

Reply:
Nah that's fine i think :)